src/lib.rs

```diff
@@ -789,6 +826,17 @@ where
     }
 }
 
+struct StaticBody {
```

Comment:
Instead of introducing a wrapper, I think we could just implement `BytesBody` for `Bytes` itself?